src/ontodia/workspace/workspaceMarkup.tsx

```diff
@@ -148,9 +165,7 @@ export class WorkspaceMarkup extends React.Component<WorkspaceMarkupProps, {}> {
                     preventTextSelection: true,
                     verticalResizing: true,
                 })}>
-                    <AccordionItem heading='Connections' bodyClassName='link-types-toolbox'>
-                        <LinkTypesToolbox view={this.props.view} editor={this.props.editor} />
-                    </AccordionItem>
+                    {items}
```

Comment:
I suggest to inline `AccordionItem` declarations here (and use conditional operator to render/not render search panel) -- this usually helps to understand the markup structure.